linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/CoreJSLib.scala

```diff
@@ -1291,58 +1290,6 @@ private[emitter] object CoreJSLib {
       })
     }
 
-    private def defineTypedArrayConversions(): Unit = {
```

Comment:
IMO this removal, and the corresponding inlining in `FunctionEmitter`, should be done in a separate commit. First only refactoring (with `FunctionEmitter` using `genCallHelper`), then doing the inlining.